demos/functions/index.js

```diff
@@ -28,7 +28,7 @@ app.set('view engine', 'hbs');
 app.get('/', function(req, res) {
   let moduleData = workboxModules.map((moduleName) => {
     const docPath = path.join(
-      __dirname, 'views', 'demo', `${moduleName}.hbs`
+        __dirname, 'views', 'demo', `${moduleName}.hbs`
```

Comment:
Idiomatic Google JS style wouldn't format the following like this:

```js
const docPath = path.join(
    __dirname, 'views', 'demo', `${moduleName}.hbs`
);
```

It'd do it like this:

```js
const docPath = path.join(
    __dirname, 'views', 'demo', `${moduleName}.hbs`);
```

The 4-space indentation is really intended to signify a line continuation, not for block indentation (which putting the closing parens on the next line makes this feel like).

That being said, I don't want to go though and fix all of these, so I'm fine with just doing what the autoformatting does for now and dealing with individual cases like this per-instance going forward.